benchmarks/IO.py

```diff
@@ -7,51 +7,78 @@
 
 TYPES = ('int64', 'float64')
 
-def time_ak_write_read(N_per_locale, trials, dtype, path, seed):
-    print(">>> arkouda {} write/read".format(dtype))
+def time_ak_write(N_per_locale, numfiles, trials, dtype, path, seed, parquet):
+    print(">>> arkouda {} write".format(dtype))
     cfg = ak.get_config()
     N = N_per_locale * cfg["numLocales"]
-    print("numLocales = {}, N = {:,}".format(cfg["numLocales"], N))
+    print("numLocales = {}, N = {:,}, filesPerLoc = {}".format(cfg["numLocales"], N, numfiles))
     if dtype == 'int64':
         a = ak.randint(0, 2**32, N, seed=seed)
     elif dtype == 'float64':
         a = ak.randint(0, 1, N, dtype=ak.float64, seed=seed)
-     
+    
     writetimes = []
+    for i in range(trials):
+        for j in range(numfiles):
+            start = time.time()
+            a.save(f"{path}{j:04}") if not parquet else a.save_parquet(f"{path}{j:04}")
+            end = time.time()
+            writetimes.append(end - start)
+    avgwrite = sum(writetimes) / trials
+
+    print("write Average time = {:.4f} sec".format(avgwrite))
+
+    nb = a.size * a.itemsize * numfiles
+    print("write Average rate = {:.2f} GiB/sec".format(nb/2**30/avgwrite))
+
+def time_ak_read(N_per_locale, numfiles, trials, dtype, path, seed, parquet):
+    print(">>> arkouda {} read".format(dtype))
+    cfg = ak.get_config()
+    N = N_per_locale * cfg["numLocales"]
+    print("numLocales = {}, N = {:,}, filesPerLoc = {}".format(cfg["numLocales"], N, numfiles))
+    a = ak.array([])
+    
     readtimes = []
     for i in range(trials):
         start = time.time()
-        a.save(path)
-        end = time.time()
-        writetimes.append(end - start)
-        start = time.time()
-        b = ak.load(path)
+        a = ak.load(path+'*') if not parquet else ak.read_parquet(path+'*')
```

Comment:
In general, `ak.load` is not supposed to take wildcards. This always trips me up, but if an array was saved with `a.save(path)`, then it should be loaded with `ak.load(path)` -- the exact same argument -- otherwise it might do crazy things. If a wildcard is necessary, it should be used with `ak.read_all()` or `ak.read_hdf()`.